projects/kit/components/items-with-more/items-with-more.service.ts

```diff
@@ -89,4 +93,19 @@ export class TuiItemsWithMoreService extends Observable<number> {
 
         return -1;
     }
+
+    private getMultilineOverflowIndex(): number {
```

Comment:
Let's rename to `getOverflowIndexMultiline` so that you can tell at a glance they are two versions of the same thing